Data.R

```diff
@@ -1,97 +1,131 @@
+#'---
+#' title: "[TRIPOD 1] Identify the study as developing and/or validating a multivariable prediction model, the target population, and the outcome to be predicted."
+#' author: 'Author One ^1,✉^, Author Two ^1^'
+#' abstract: |
+#'  | [TRIPOD 2] Provide a summary of objectives, study design, setting, participants, sample size, predictors, outcome, statistical analysis, results, and conclusions.
+#' documentclass: article
+#' description: 'Manuscript'
+#' clean: false
+#' self_contained: true
+#' number_sections: false
+#' keep_md: true
+#' fig_caption: true
+#' css: production.css
+#' output:
+#'   html_document:
+#'    code_folding: hide
+#'    toc: true
+#'    toc_float: true
+#' ---
+#'
 #' Loading R libraries
+#+ init, echo=FALSE, message=FALSE, warning=FALSE
+debug <- 0;
+knitr::opts_chunk$set(echo=debug>-1, warning=debug>0, message=debug>0, max.print=42);
 library(ggplot2)
 library(dplyr)
 library(GGally)
 library(rio)
 library(pander)
 library(maps)
+library(printr)
+options(max.print=42)
+panderOptions('table.split.table',Inf); panderOptions('table.split.cells',Inf);
 #' Create data set
 dat0 <- import("covid19cleanedamericas.csv", stringsAsFactors=F);
 #' base R scatterplot matrix
+#+ scatterplot, cache=TRUE
```

Comment:
In parts of your script that take a long time to run, I added a line starting with `#+` that gives the subsequent code chunk a name and caches its results the first time it runs so it only needs to re-run if the data is changed or you delete the cache. This only affects reports, not running it in console, though.